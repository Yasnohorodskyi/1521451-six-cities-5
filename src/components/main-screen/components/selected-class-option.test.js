import React from "react";
import renderer from "react-test-renderer";

import SelectedClassOption from "./selected-class-option";
import withOpenSelect from '../../../hocs/with-open-select';

const SelectedClassOptionWithClickOutside = withOpenSelect(SelectedClassOption);

it(`Render SelectedClassOption`, () => {
  const tree = renderer
    .create(
        <SelectedClassOptionWithClickOutside />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
