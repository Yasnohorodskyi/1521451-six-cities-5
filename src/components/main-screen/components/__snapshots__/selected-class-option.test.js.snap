// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SelectedClassOption 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
  onClick={[Function]}
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
  <span
    className="places__sorting-type"
    data-action="open"
    tabIndex="0"
  >
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom places__options"
  >
    <li
      className="places__option "
      data-action="change"
      tabIndex="0"
    >
      Popular
    </li>
    <li
      className="places__option "
      data-action="change"
      tabIndex="0"
    >
      Top rated first
    </li>
    <li
      className="places__option "
      data-action="change"
      tabIndex="0"
    >
      Price: low to high
    </li>
    <li
      className="places__option "
      data-action="change"
      tabIndex="0"
    >
      Price: high to low
    </li>
  </ul>
  <select
    className="places__sorting-type"
    defaultValue="Popular"
    id="places-sorting"
  >
    <option
      className="places__option"
      value="popular"
    >
      Popular
    </option>
    <option
      className="places__option"
      value="popular"
    >
      Top rated first
    </option>
    <option
      className="places__option"
      value="popular"
    >
      Price: low to high
    </option>
    <option
      className="places__option"
      value="popular"
    >
      Price: high to low
    </option>
  </select>
</form>
`;
